packages/graph-explorer/src/modules/CreateConnection/CreateConnection.tsx

```diff
@@ -271,19 +275,32 @@ const CreateConnection = ({
           </div>
         )}
         {form.proxyConnection && form.awsAuthEnabled && (
-          <div className={pfx("input-url")}>
-            <Input
-              data-autofocus={true}
-              label={"AWS Region"}
-              value={form.awsRegion}
-              onChange={onFormChange("awsRegion")}
-              errorMessage={"Region is required"}
-              placeholder={"us-east-1"}
-              validationState={
-                hasError && !form.awsRegion ? "invalid" : "valid"
-              }
-            />
-          </div>
+          <>
+            <div className={pfx("input-url")}>
+              <Input
+                data-autofocus={true}
+                label={"AWS Region"}
+                value={form.awsRegion}
+                onChange={onFormChange("awsRegion")}
+                errorMessage={"Region is required"}
+                placeholder={"us-east-1"}
+                validationState={
+                  hasError && !form.awsRegion ? "invalid" : "valid"
+                }
+              />
+            </div>
+            <div className={pfx("input-url")}>
+              <Select
+                label={"Service Type"}
+                options={[
+                  { label: "Neptune DB", value: "neptune-db" },
```

Comment:
Will this value be picked as a default selection?

Reply:
yes